### PR TITLE
Successfully running TiJepsen

### PR DESCRIPTION
**Pull request by UncP**

### CODE REVIEW

File: tidb/project.clj [hunk 1]
```diff
@@ -5,6 +5,6 @@
             :url "http://www.eclipse.org/legal/epl-v10.html"}
   :main tidb.core
   :dependencies [[org.clojure/clojure "1.8.0"]
-                 [jepsen "0.1.6"]
+                 [jepsen "0.1.5"]
```
Comment (siddontang) [inline tidb/project.clj]: why decrease the version?

Comment (UncP) [inline tidb/project.clj]: unable to find version 0.1.6 in clojure package manager

Comment (UncP) [inline tidb/project.clj]: I will try 0.1.6 next time to see if it's ok.

File: tidb/src/tidb/db.clj [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+(defn initial-cluster                      
+  "Constructs an initial cluster string for a test, like \"foo=foo:2380,bar=bar:2380,...\""                                                                                  
+  [test]                                   
+  (->> (:nodes test)                       
+       (map (fn [node] (str (get-in tidb-map [node :pd]) "=" (peer-url node))))       
+       (str/join ",")                      
+  )                                        
+)                                          
+
+(defn pd-cluster
```
Comment (siddontang) [inline tidb/src/tidb/db.clj]: pd-endpoints

File: tidb/project.clj (hunk 1 above)
Comment (siddontang) [inline tidb/project.clj]: any update?

Comment (UncP) [inline tidb/project.clj]: tried again, same issue.

File: tidb/src/tidb/db.clj [hunk 3]
```diff
@@ -165,19 +159,21 @@
...
+        (sql! "create database if not exists jepsen;")
+        (sql! (str "GRANT ALL PRIVILEGES ON jepsen.* TO \'jepsen\'@\'%\' IDENTIFIED BY \'jepsen\';"))
       )
     )
     (teardown! [_ test node]
       (info node "tearing down TiDB")
       (cu/stop-daemon! tidbbin dbpidfile)
       (cu/stop-daemon! tikvbin kvpidfile)
       (cu/stop-daemon! tipdbin pdpidfile)
+      (cu/grepkill! "mysqld")
```
Comment (siddontang) [inline tidb/src/tidb/db.clj]: why mysql here?

**Pull request closed without merging**